src/pages/Home/HomePage.tsx

```diff
@@ -16,7 +29,11 @@ export const HomePage = (): JSX.Element => {
         <HistorySection />
         <Divider />
         <GoalSection />
-        <Carousel />
+        <StyledCarousel
+          showSingleSlide={platform !== Platform.Web}
```

Comment:
On mobile, images are so large and they are breaking layout, please add more styles. Also:
```suggestion
        <StyledCarousel
          showSingleSlide={platform === Platform.Mobile}
```